src/patchers/UnaryExistsOpPatcher.js

```diff
@@ -0,0 +1,24 @@
+import UnaryOpPatcher from './UnaryOpPatcher.js';
+
+export default class UnaryExistsOpPatcher extends UnaryOpPatcher {
+  patchAsExpression() {
+    const node = this.node;
+    const nodeExpression = node.expression;
+
+    // Only add parentheses if necessary
+    if (node.parentNode && node.parentNode.type !== 'Block') {
+        this.insertBefore('(');
+        this.insertAfter(')');
+    }
+
+    if (nodeExpression && nodeExpression.type === 'Identifier') {
+      this.overwrite(this.start, this.end, `typeof ${nodeExpression.raw} !== 'undefined' && ${nodeExpression.raw} !== null`);
+    } else {
+      this.overwrite(this.expression.after, this.end, ' != null');
```

Comment:
I've started including comments here to explain more clearly what's going on. Here I'd put this:

```
// `a?` → `a != null`
//   ^      ^^^^^^^^
```

And same for the `Identifier` case.